CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ zephyr_include_directories(${CMAKE_SOURCE_DIR})
 project(bloodlight_rev2)
 
 target_sources(app PRIVATE 
-    src/bl.c
```

Comment:
I don't think this commit has anything to do with the rest of the changes. We should probably create another pull request to fix the build of bl.c.